cmd/unity/cmd/project.go

```diff
@@ -582,6 +586,23 @@ func runModule(log io.Writer, info runModuleInfo) (err error) {
 			// Limit concurrency across all testscript runs
 			// e.Defer(m.tester.limit())
 
+			// getVal is a helper function to lookup an environment
+			// variable value within e.Vals
+			getVal := func(k string) string {
```

Comment:
or use https://pkg.go.dev/github.com/rogpeppe/go-internal/testscript#Env.Getenv :)

Reply:
You just know I'd forgotten about these :)

Done.